### DISCUSSION TITLE
Merging Azure Wiki pages?

### DISCUSSION THREAD

**Discussion by blindzero** [OpenIDC/mod_auth_openidc#508] [Ideas]
Hi,

first of all: thanks for this great module, it really saved my life ;)
However, first I had some issues to get it running with Azure AD, even with a simple test page.
On the one hand maybe a little lack of knowledge on my side. But on the other hand I found the Wiki pages a little confusing as there are two:
* https://github.com/zmartzone/mod_auth_openidc/wiki/Azure-OAuth2.0-and-OpenID
* https://github.com/zmartzone/mod_auth_openidc/wiki/Azure-OAuth-2.0-and-OpenID-Connect

I understand that OpenID and OpenID Connect technically are two different things, but wouldn't it make sense to merge those pages here? As far as I undersand the first page covers the Azure config part and the second the module configuration for Azure?

What do you think? I am happy to do this but wanted to discuss first.

Thanks,
Matthias

**Comment by zandbelt** [comment]
sure, you can merge if you think that's better (I'm not qualified to judge right now)

**Reply by blindzero** [reply]
Did it: https://github.com/zmartzone/mod_auth_openidc/wiki/Azure-Active-Directory-Authentication

**Reply by blindzero** [reply]
Also updated README.md and sent PR #509
